gef-ui/server/controllers/ukef-cover-start-date/index.js

```diff
@@ -0,0 +1,96 @@
+const { format } = require('date-fns');
+const
+  {
+    validationErrorHandler,
+    pastDate,
+    futureDateInRange,
+    getEpoch,
+  } = require('../../utils/helpers');
+const { applicationDetails } = require('../application-details');
+const api = require('../../services/api');
+const CONSTANTS = require('../../../constants');
+
+const setError = (field, message) => validationErrorHandler({
+  errRef: field,
+  errMsg: message,
+});
+
+const updateCoverStartDate = async (facilityId, { coverStartDate, shouldCoverStartOnSubmission }) => {
+  try {
+    return api.updateFacility(facilityId, {
+      coverStartDate,
+      shouldCoverStartOnSubmission,
+      coverDateConfirmed: true,
+    });
+  } catch (error) {
+    console.error('Unable to update the facility.', { error });
+  }
+  return false;
+};
+
+const processCoverStartDate = async (req, res) => {
+  const { applicationId, facilityId } = req.params;
+  const {
+    ukefCoverStartDate,
+    day,
+    month,
+    year,
+  } = req.body;
+  let facility;
+
+  try {
+    if (ukefCoverStartDate === undefined) {
```

Comment:
Could it be this instead?

```js
if (!ukefCoverStartDate) {
```

Reply:
No I'm afraid as this will trigger false branch as well.